src/lib/FlagProvider.svelte

```diff
@@ -1,12 +1,12 @@
 <script lang="ts">
 	import { setContext, onMount } from 'svelte';
-	import { ContextStateSymbol, type TContext, type eventArgs } from './context';
+	import { ContextStateSymbol, type TContext, type eventArgs } from './context.js';
```

Comment:
I was going to ask about adding .js but found the answer: https://kit.svelte.dev/docs/packaging#caveats